src/m365/spo/commands/serviceprincipal/serviceprincipal-permissionrequest-list.spec.ts

```diff
@@ -17,6 +17,95 @@ describe(commands.SERVICEPRINCIPAL_PERMISSIONREQUEST_LIST, () => {
   let log: string[];
   let logger: Logger;
   let loggerLogSpy: sinon.SinonSpy;
+  const oAuth2PermissiongrantsResponse = {
```

Comment:
Let's consider permission grants two words as they are and rename to `oAuth2PermissionGrantsResponse`